### PR TITLE
[zh]Sync content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 1]
```diff
@@ -312,13 +312,13 @@ runtime where possible.
...
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参阅[从 docker cli 到 crictl 的映射](/zh/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl))。
+(参阅[从 docker cli 到 crictl 的映射](/zh/docs/tasks/debug/debug-cluster/crictl/crictl/#mapping-from-docker-cli-to-crictl))。
 对于后者，可以用新的容器创建选项，例如
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md:322]: Another PR is fixing the same file with new section added.

Comment (kinzhi) [reply content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md]: Yes, I resolved the conflict

File: content/zh/blog/_posts/2020-12-02-dockershim-faq.md [hunk 2]
```diff
@@ -267,13 +267,13 @@ runtime where possible.
...
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
+(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-12-02-dockershim-faq.md:276]:
```suggestion
(参见 [从 docker 命令映射到 crictl](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```

**Comment by kinzhi** [comment]
Please help to review this PR. @tengqm

**Comment by mengjiao-liu** [comment]
Please `squeeze` commits and resynchronize file `content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md`.

File: content/zh/blog/_posts/2020-12-02-dockershim-faq.md [hunk 3]
```diff
@@ -267,13 +267,13 @@ runtime where possible.
...
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
+(参见 [从 docker 命令映射到 crictl](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-12-02-dockershim-faq.md:276]:
```suggestion
(参见[从 docker 命令映射到 crictl](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```

**Comment by tengqm** [comment]
Rather than having contributors randomly pick a page to sync, please check the issues #33474, #33475, #33476, #33477, #33478, #33479 and sign on tasks there. It would help save your time and avoid conflicts when working on different pages.

**Comment by tengqm** [comment]
Please don't mix two languages in one PR.

**Comment by kinzhi** [comment]
> Please don't mix two languages in one PR.

Ok, I've split up the other languages pr

**Comment by tengqm** [comment]
The `troubleshooting-kubeadm.md` file has more changes to be sync'ed than the ones carried by this PR.

**Comment by kinzhi** [comment]
> The `troubleshooting-kubeadm.md` file has more changes to be sync'ed than the ones carried by this PR.

Ok, I will resubmit the PR

**Pull request closed without merging**